omero/developers/JavaGateway.txt

```diff
@@ -0,0 +1,20 @@
+Java Gateway
+------------
+
+The Java :javadoc:`Gateway <omero/gateway/Gateway.html>` is a
+wrapper around the :zerocdoc:`Ice Java language mapping
+<display/Ice/Hello+World+Application>` and the :doc:`Modules/Api` which makes
+it easier to interact with an OMERO server in Java.
+
+The :javadoc:`Gateway <omero/gateway/Gateway.html>` is the central object
+for maintaining the connection to the server. 
+See :ref:`gatewayconnect`
+
+Functionality for interacting with the server is encapsulated into different
+:javadoc:`Facilities </omero/gateway/facility/package-summary.html>`
+For example using the :javadoc:`BrowseFacility <omero/gateway/facility/BrowseFacility.html>`
```

Comment:
you're got a capital F here but this isn't really a new sentence, use e.g. and carry on the current sentence instead?